src/components/assets/Account.vue

```diff
@@ -290,6 +276,8 @@ export default defineComponent({
 
     const currentNetworkName = ref<string>(providerEndpoints[currentNetworkIdx.value].displayName);
 
+    const { isDappStakingV3 } = useDappStaking();
```

Comment:
Do you need this line in this file? It seems `isDappStakingV3` hasn't been used.

Reply:
@impelcrypto thank you! I forgot to remove it. Fixed in [677420c](https://github.com/AstarNetwork/astar-apps/pull/1149/commits/677420c5de7c1916855cbd9f2ed1819cd2bd9fc9)